### PR TITLE
Some optimizations arjenvaneerde

### PR DESCRIPTION
**Pull request by arjenvaneerde**
#### Check List:
- [x] Tests pass (`./test.sh <username>` shows no differences between expected and actual outputs)
- [x] All formatting changes by the build are committed
- [x] Your launch script is named `calculate_average_<username>.sh` (make sure to match casing of your GH user name) and is executable
- [x] Output matches that of `calculate_average_baseline.sh`
* Execution time: 13.4 sec
* Execution time of reference implementation: 5m45s

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Hey, could you rebase this one to latest main and squash everything into one commit? I feel I've messed handling the initial PR, as somehow the file seems gone. But we can add it back with this one. Thx!

**Comment by arjenvaneerde** [comment]
Hi Gunnar, I've rebased my changes on your main, however I don't seem to be able (using IntelliJ) to squash commits. I did find that commits could be squashed when merging this PR, but that is probably an option you have to select. Could also be my lack of Git knowledge.

File: calculate_average_arjenvaneerde.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+
+JAVA_OPTS=""
+time java $JAVA_OPTS --class-path target/average-1.0.0-SNAPSHOT.jar dev.morling.onebrc.CalculateAverage_arjenvaneerde
```
Comment (gunnarmorling) [inline calculate_average_arjenvaneerde.sh:20]:
```suggestion
java $JAVA_OPTS --class-path target/average-1.0.0-SNAPSHOT.jar dev.morling.onebrc.CalculateAverage_arjenvaneerde
```

**Comment by gunnarmorling** [comment]
00:11.934. Thx for particpating!

**Pull request merged**